h2o-automl/src/main/java/water/automl/api/schemas3/AutoMLBuildSpecV99.java

```diff
@@ -142,7 +142,7 @@
             level = API.Level.secondary)
     public int max_models;
 
-    @API(help = "Maximum time to spend building all models (optional).",
+    @API(help = "Maximum time to spend building all models. Defaults to 1h if max_models is not provided. (optional).",
```

Comment:
@seb-h2o should we say `optional` if we have already said `Defaults to...`?

Reply:
It doesn't hurt I guess.
I'm not even sure we're always consistent with those "optional" info.

Reply:
The "(optional)" thing is an old R convention.  We are not consistent about it and I think having it there only part of the time is more confusing than not having it there at all.  At some point, we should just go through and remove it everywhere.